docs/pages/cli/verify.mdx

```diff
@@ -0,0 +1,61 @@
+# mud verify
+
+This command uploads the source code for the deployed contracts to a public repository, such as a block explorer.
+The repository then compiles the source code and verifies that the compiled version is identical to what is deployed on-chain.
+
+## Using the command
+
+1. Before you run `mud verify` you need to create a [Foundry profile](https://book.getfoundry.sh/reference/config/overview).
+   Change to the `packages/contract` directory and edit `foundry.toml` to add a profile with the `eth_rpc_url` for your network (if needed).
+   For example, add these lines to add Holesky.
+
+   ```toml copy
+   [profile.holesky]
+   eth_rpc_url = "https://ethereum-holesky.publicnode.com"
+   ```
+
+   Note that this blockchain has to be supported by the verifier.
+
+1. If using Blockscout, you need to specify the API key:
+
+   ```sh copy
+   export ETHERSCAN_API_KEY=<value goes here>
+   ```
+
+1. Run the command.
```

Comment:
I think it's worth noting here that verify will only work with the original source + compiler options + compiled artifacts all match the onchain bytecode. So verifying should often happen immediately after deploying and before any further edits, package updates, etc. are done.